rfcs/testdriver_set_time_zone.md

```diff
@@ -0,0 +1,21 @@
+# RFC 73 : Add testdriver.js support for the `timeZone` WebDriver command.
+
+## Summary
+Add testdriver.js support for the `timeZone` WebDriver extension, which will
+allow setting the current timezone in the browser. This extension is specced as
+part of the [HTML spec](https://github.com/whatwg/html/pull/3047).
```

Comment:
The proposed text is currently in this comment: https://github.com/whatwg/html/pull/3047#issuecomment-732158684

This RFC should not be merged until the comment text is (a) part of the PR, and (b) the PR is either landed or fully LGTM'd and about to be landed.

Reply:
My comment text is now part of the PR: https://github.com/whatwg/html/pull/3047